BlueGreenDeploymentOrchestrator/generator-function.js

```diff
@@ -1,10 +1,18 @@
 const moment = require('moment');
 
+const checkMandatoryInput = (input, parameterName) => {
```

Comment:
Could this be pulled out into its own file?

Reply:
Did consider this but felt that the code had been extracted to a function for readability rather than utility so discounted it. If more 'rules' are added (e.g. checking type) then would opt to move it into a separate file and add unit tests or opt to use an external library for value/type checking (e.g. `assert-args`).